rep-0105.rst

```diff
@@ -54,6 +54,8 @@ The ``map`` frame is useful as a long-term global reference, but
 discrete jumps make it a poor reference frame for local sensing and
 acting.
 
+It is recommended to align the map with east and north for easier viewing,
+but is not required.
```

Comment:
This wording is a bit vague to me. After a bit of consideration, I concluded that an East-North-Up (ENU) orientation is being recommended, but not required. If that is the case, why not say so?